LICENSE.txt

```diff
@@ -187,6 +187,7 @@
       identification within third-party archives.
 
    Copyright 2020 Real Folk Inc.
+   Copyright 2022 Province of British Columbia
```

Comment:
I think copyright years should be 2019 for Real Folk (started SWU contract in 2019) and 2020 for govt (Real Folk worked for BCGov under a general IT services agreement in 2020).